Clear error messages on verify message page

tuhalang opened the pull request:
Resolves #392 
- This modifies the error messages clear after correct values have been inputted and the verify message button is clicked.

metaclips commented on fyne/pages/staking.go:
```diff
@@ -0,0 +1,143 @@
+	}
+
+	if staking.stakeInfoLabel.Text == "" {
+		staking.stakeInfoLabel.Text = stakeInfoText
```
If you aren't using SetText you should call widget refresh.

itswisdomagain replied:
If I use `SetText`, the program crashes because the label has not been added to the UI at this point.
If the text in the label is empty, then it has not been added to the page. Setting the `Text` property directly works without requiring a refresh.

metaclips replied:
Yes, this has been fixed, please check https://github.com/fyne-io/fyne/issues/381

metaclips replied:
add a layout.Spacer() before and after the submit button in a NewHBox, that should center it.

tuhalang committed:
Clear error messages on verify message page

- This modifies the error messages clear after correct values have been inputted and the verify message button is clicked.

Sirmorrison reviewed (dismissed):
PR test and functionality looks good.

tuhalang committed:
Increase the copy button size
